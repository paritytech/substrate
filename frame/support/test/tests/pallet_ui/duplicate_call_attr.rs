#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::Hooks;
	use frame_system::pallet_prelude::BlockNumberFor;
	use frame_support::pallet_prelude::StorageValue;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(core::marker::PhantomData<T>);

	#[pallet::hooks]
	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}

	#[pallet::call]
	impl<T: Config> Pallet<T> {}

	#[pallet::storage]
	type Foo<T> = StorageValue<_, u8>;

	#[pallet::call]
	impl<T: Config> Pallet<T> {}
}

fn main() {
}
