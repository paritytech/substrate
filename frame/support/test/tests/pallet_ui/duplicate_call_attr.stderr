error: Invalid duplicated attribute
  --> $DIR/duplicate_call_attr.rs:22:12
   |
22 |     #[pallet::call]
   |               ^^^^
